_config.yml

```diff
@@ -277,6 +277,11 @@ mathjax:
 # Gentie productKey
 #gentie_productKey:
 
+# support youyan comments,
+# you can regist and get your uid from http://www.uyan.cc
+# youyan uid ,like youyan_id=2343223
```

Comment:
Fix on this:
# Support for youyan comments system.
# You can get your uid from http://www.uyan.cc